src/app/pages/product/product-page.module.ts

```diff
@@ -60,5 +61,6 @@ const productPageRoutes: Routes = [
     ProductPageComponent,
     RetailSetPartsComponent,
   ],
+  exports: [ProductDetailInfoComponent],
```

Comment:
exports is not necessary because you use it only on (within) product detail page